resources/schema/spidamin/spidadb/referenced_design.schema

```diff
@@ -91,6 +91,18 @@
         "clientFileId": {
             "type": "string",
             "description": "The unique id of this minimal client data object in mongo"
+        },
+        "status": {
+            "type": "string",
+            "description": "ACTIVE: Design has been promoted to the active design, INACTIVE: Design was promoted previously, but a new design has been promoted over it, (not present): Design is either associated with a PM Project or from an older version of studio."
+        },
+        "sourceProjectLabel": {
+            "type": "string",
+            "description": "The label of the project that an ACTIVE or INACTIVE design was originally from."
+        },
+        "sourceProjectId": {
+            "type": "string",
+            "description": "The id (in SPIDAdb) of the project that an ACTIVE or INACTIVE design was originally from."
         }
     },
     "additionalProperties": false
```

Comment:
Since additionalProperties is false this will need a changeset to remove these fields when the json is downconverted.

Reply:
calc schemas are validated in studio then placed in the "referenced" schemas as "calcLocation", "calcDesign" etc., the "referenced" schemas changed here are for spida db and used in studio; studio gets/puts these to mongodb as BSON format (and converted to Map) and aren't validated as JSON; there was a similar addition to the "referenced" schema in 2019 without any up/down conversion (client file id)